tck-runner/pom.xml

```diff
@@ -323,5 +320,47 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>security-manager</id>
+            <activation>
+                <property>
+                    <name>with-security-manager</name>
+                    <value>true</value>
+                </property>
+            </activation>
+            <properties>
+                <arquillian.protocol>LocalSecurityManagerTesting</arquillian.protocol>
```

Comment:
Yes, it is a custom protocol, essentially it's the same as "Local", only that it puts one frame onto the stack which we can assign with minimum rights in the test.policy so to discover any missing privileged blocks in HV. I don't have any documentation of this at hand unfortunately.